alpha/action/migrate_test.go

```diff
@@ -16,6 +17,13 @@ import (
 	"github.com/operator-framework/operator-registry/pkg/lib/bundle"
 )
 
+type migrationLevel string
+
+const (
+	bundleObjectType migrationLevel = "bundle-object"
```

Comment:
This is basically "no migrations", right?

I think we talked about this, but I feel like these tests could get very cumbersome once we have more than 2 migrations.

What if, instead, these tests ran all the migrations, and then we have a separate set of unit tests for the `migrations` package?

Reply:
(actually see my other comment about putting `*migrations.Migrations` on these structs and then just defining a dummy test `Migrations` instead)

Reply:
There's a nuance here which is not the same as elsewhere.  I agree that we ought to be clear that what we're testing here is all/nothing and not incremental.  But in this case these are just coincidentally using identifiers that align with the current [0...1] options. 
We're using it in testdata construction to align with the selected all/none migrations. 
I've renamed/retyped this area to make it hopefully more clear.  WDYT?

Reply:
Adjusted this as we discussed to 
1. test only the none/arbitrary migrations cases, to verify that migrations-as-a-mechanism works;
2. introduce unit tests in actions.migrations which allow us to evaluate our sequence of migrators to protect against surprises along the processing chain